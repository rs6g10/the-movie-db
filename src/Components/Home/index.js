import React from 'react';
import Header from '../Header';

const API_KEY = process.env.API_KEY;

const Home = () => (
  <div className="container">
    <div className="row">
      <div className="col-xs-12 col-lg-10 offset-lg-1">
        <Header apiKey={API_KEY} />
      </div>
    </div>
  </div>
);

export default Home;
