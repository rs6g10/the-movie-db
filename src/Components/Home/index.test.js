import '../../test/setupEnzyme';

import React from 'react';
import { shallow } from 'enzyme';

import Home from './';

describe('Home', () => {
  const props = { apiKey: 'blah' };

  const element = shallow(<Home {...props} />);

  it('should render as expected', () => {
    expect(element).toMatchSnapshot();
  });
});
