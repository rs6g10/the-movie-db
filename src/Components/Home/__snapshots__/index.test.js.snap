// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render as expected 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-lg-10 offset-lg-1"
    >
      <Header
        apiKey="MYSUPERSECRETKEY"
      />
    </div>
  </div>
</div>
`;
